std/hash/README.md

```diff
@@ -0,0 +1,54 @@
+# std/hash
+
+## MD5
+
+**Uses:**
+
+```ts
+import { Md5 } from "https://deno.land/std/hash/md5.ts";
+
+const md5 = new Md5();
+const md5Instance = md5.update(millionAs); // return instance of `Md5`
+console.log(md5Instance instanceof Md5); // true
+console.log(md5Instance.toString()); // 7707d6ae4e027c70eea2a935c2296f21
+```
+
+Calling `update` method, It will update internal state based on the input
+provided. Once you call `md5Instance.toString()`, it will return the
+`hash string`. You can provide format as `hash` or `base64`. The default format
+is `hex`.
+
+**sample:**
+
+```ts
+console.log(md5Instance.toString("base64")); // RWYpygjdUY/oeXcpO7aNqg==
+```
+
+## SHA1
+
+**Uses:**
+
+```ts
+import { Sha1 } from "https://deno.land/std/hash/sha1.ts";
+
+const sha1 = new Sha1().update("中文");
+console.log(sha1.hex()); // 7be2d2d20c106eee0836c9bc2b939890a78e8fb3
+console.log(sha1.toString()); // default is hex, 7be2d2d20c106eee0836c9bc2b939890a78e8fb3
+```
+
+## Sha256 and HmacSha256
+
+**Uses:**
+
+```ts
+import { Sha256, HmacSha256 } from "https://deno.land/std/hash/sha256.ts";
+
+const sha256 = new Sha256().update("中文");
+console.log(sha256.hex()); // 72726d8818f693066ceb69afa364218b692e62ea92b385782363780f47529c21
+console.log(sha256.toString()); // Same as above, 72726d8818f693066ceb69afa364218b692e62ea92b385782363780f47529c21
+
+const key = "Hi There";
+const hmac = new HmacSha256(key).update("中文");
+console.log(hmac.hex()); // 2f1ee7f7e89d3d53deadc1324dd0016bf8b859c620a910b3f6dba7b1a519d520
+console.log(hmac.toString()); // Same as above, 2f1ee7f7e89d3d53deadc1324dd0016bf8b859c620a910b3f6dba7b1a519d520
```

Comment:
The comments showing output are not useful

Reply:
Should I move as a statement before code? i thought it would be too verbose.

Reply:
How about this?

## Sha256 and HmacSha256

**Uses:**

Creating `Sha256` hash is simple. You can use `Sha256` class instance and update the digest. Calling `hex` method will return the sha256 in `hex` value. You can also use `toString` method.

Note: For `HmacSha256`, you can pass secret `key` while creating instance of the object.

Reply:
just remove the comments - they make the lines longer than 80 characters - and it doesn't add to the example.

Reply:
Done! Thanks.